### PR TITLE
Update for langpacks from 1.31

### PR DESCRIPTION
**Pull request by smartguest**
This PR updates the langpack source files to what was included with the 1.31 release. (XLFs are separate and already updated beyond the 1.31 release)

### CODE REVIEW

File: i18n/ads-language-pack-de/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	"license": "SEE SOURCE EULA LICENSE IN LICENSE.txt",
 	"engines": {
 		"vscode": "*",
-		"azdata": ">=1.29.0"
+		"azdata": "^0.0.0"
```
Comment (Charles-Gagnon) [inline i18n/ads-language-pack-de/package.json:14]: These should be >=1.31.0 right?

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

File: i18n/ads-language-pack-de/package.json (hunk 1 above)
Comment (smartguest) [reply i18n/ads-language-pack-de/package.json]: That is correct, I'm changing it to ^1.31.0 since that's following what vscode uses.

Comment (smartguest) [reply i18n/ads-language-pack-de/package.json]: Is it safe to create a main build then use the resulting vsixes from there to overwrite the ones in storage explorer or will this require a version bump?

Comment (Charles-Gagnon) [reply i18n/ads-language-pack-de/package.json]: Why would we need to release a new version? Go ahead and change the engine string if you want, but that kind of change doesn't have any impact so I don't think we need to update the VSIXs that are published.

**Review by Charles-Gagnon — APPROVED** [review]

File: i18n/ads-language-pack-de/package.json (hunk 1 above)
Comment (smartguest) [reply i18n/ads-language-pack-de/package.json]: It doesn't affect the fact you can install the langpack, just that some strings might not work on older versions.

**Pull request merged**